web/src/AutoBoundariesMode.svelte

```diff
@@ -108,10 +108,7 @@
         mode: "neighbourhood",
       };
     } catch (err) {
-      console.log("error when setting auto-generated neighborhood", err);
```

Comment:
No problem with this change, but I'm curious - were you able to trigger this? (How?)

Reply:
![image](https://github.com/user-attachments/assets/16c7a64a-66cd-47c7-9eae-0785ddb23ece)
Yes, just draw a boundary inside a larger residential area, and have no primary roads involved

Reply:
Ah! I think this is related to https://github.com/a-b-street/ltn/pull/216#discussion_r1988889104

And I've triggered it by having no primary roads in my hand-drawn neighbourood:
<img width="892" alt="Screenshot 2025-03-11 at 10 20 35" src="https://github.com/user-attachments/assets/ce3ef936-c466-4894-83a5-e4062b064bde" />